lib/turbograft.rb

```diff
@@ -36,7 +36,7 @@ def referer
 
       ActiveSupport.on_load(:action_view) do
         (ActionView::RoutingUrlFor rescue ActionView::Helpers::UrlHelper).module_eval do
-          include XHRUrlFor
+          prepend XHRUrlFor
```

Comment:
Thank you for such a thorough and well written explanation 😃  🎆 

Makes a lot of sense.